src/main/java/spoon/reflect/path/CtRole.java

```diff
@@ -202,10 +202,10 @@ public CtRole getSuperRole() {
 	/**
 	 * @return sub role of this role, which match `item`.
 	 *
-	 * <code><pre>
```

Comment:
why this change? this is strictly equivalent.

Reply:
The error message from IDEA code inspection is: `Block element not allowed within inline element <code>: pre`

I think it should be like here https://stackoverflow.com/questions/4611591/code-vs-pre-vs-samp-for-inline-and-block-code-snippets

```
Use <code> for inline code that can wrap and <pre><code> for block code that must not wrap. 
```